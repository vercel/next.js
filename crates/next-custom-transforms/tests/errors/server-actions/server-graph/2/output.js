/* __next_internal_action_entry_do_not_use__ {"00ac840dcaf5e8197cb02b7f3a43c119b7a770b272":"bar"} */ import { registerServerReference } from "private-next-rsc-server-reference";
import { encryptActionBoundArgs, decryptActionBoundArgs } from "private-next-rsc-action-encryption";
'use strict';
export function bar() {}
import { ensureServerEntryExports } from "private-next-rsc-action-validate";
ensureServerEntryExports([
    bar
]);
registerServerReference(bar, "00ac840dcaf5e8197cb02b7f3a43c119b7a770b272", null);
