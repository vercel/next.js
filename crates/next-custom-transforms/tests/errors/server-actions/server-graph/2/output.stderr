  x Server Actions must be async functions.
  | 
   ,-[input.js:7:1]
 6 | 
 7 | export function bar() {}
   :        ^^^^^^^^^^^^^^^^^
   `----
